readme.txt

```diff
@@ -90,6 +90,9 @@ The Currency Switcher plugin is just an addon to help users get an idea of how m
 
 == Changelog ==
 
+= 1.0.4 =
+* Improvement: Escaped URLs wherever required.
```

Comment:
Changelog should be `Improvement: esc_url_raw() applied to prevent potential XSS issues.`